CMakeLists.txt

```diff
@@ -101,8 +101,8 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT AND CMAKE_PROJECT_NAME STREQUAL P
 endif()
 
 # Check to make sure the install prefix isn't the build folder, if it is, build errors will happen
-get_filename_component(tmp_install_prefix ${CMAKE_INSTALL_PREFIX} REALPATH)
-get_filename_component(tmp_proj_bindir ${PROJECT_BINARY_DIR} REALPATH)
+get_filename_component(tmp_install_prefix "${CMAKE_INSTALL_PREFIX}" REALPATH)
```

Comment:
Should we have a specific warning in this case?  Seems like in most cases if the install prefix is empty that represents some problem with the command line or something of that nature.

Reply:
I think that's true. In almost all cases its probably that they used an empty environment variable either from a typo or that they forgot to set.